build-system/tasks/dep-check.js

```diff
@@ -78,6 +78,9 @@ function Rule(config) {
 
   /** @private @const {!Array<string>} */
   this.whitelist_ = toArrayOrDefault(config.whitelist, []);
+
+  /** @const {!Set<string>} */
+  this.unusedAllowlistEntries = new Set(this.whitelist_);
```

Comment:
+1 to using "allowlist" over "whitelist". Would it be easy to do a case-sensitive find-replace of all instances in this file?

Reply:
Awesome, I'll update it. I'd love to do this everywhere in amphtml.

Reply:
Thanks for doing this. Note though, that the replacement needs to be case sensitive.